app/src/main/res/values/strings.xml

```diff
@@ -174,20 +174,20 @@
     <!-- error dialogs -->
     <string name="dialog_error_details_button_default">Show details</string>
     <string name="dialog_error_details_default">No details available</string>
-    <string name="dialog_error_details_write">This station is probably a duplicate or else you might want to check your external storage.</string>
+    <string name="dialog_error_details_write">Check your external storage if this station isn't a duplicate.</string>
```

Comment:
What would one check external storage for?

Reply:
There could be two reasons why the write fails: 1. Duplicate File 2. External Storage unmounted (unlikely). The string is shown when a user taps on "Show details". In an error dialog. I think "Station is probably a duplicate." might be good here.